packages/kit/src/views/Onboardingv2/components/PhaseInputArea.tsx

```
@@ -495,7 +504,9 @@ export function PhaseInputArea({
     const { mnemonicType } = await serviceAccount.validateMnemonic(
       mnemonicEncoded,
     );
-    onConfirm({ mnemonic: mnemonicEncoded, mnemonicType });
+    const result = { mnemonic: mnemonicEncoded, mnemonicType }; 
```

ESLint

[error] Delete `·`

(prettier/prettier)